.buildkite/package.json

```
@@ -26,7 +26,7 @@
     "targz": "^1.0.1",
     "tmp": "^0.2.1",
     "ts-node": "^10.9.2",
-    "typescript": "^5.5.3",
+    "typescript": "^5.6.3",
```

Can you upgrade to `^6`?

```suggestion
    "typescript": "^6.0.0",
```